src/__internal__/sticky-footer/sticky-footer.style.js

```diff
@@ -1,29 +1,26 @@
 import styled, { css, keyframes } from "styled-components";
-import baseTheme from "../../style/themes/base";
 
 const FormButtonAnimation = keyframes`
   0%   { transform: translateY(50px); }
   100% { transform: translateY(0px); }
 `;
 
 const StyledStickyFooter = styled.div`
-  ${({ sticky, theme }) => css`
+  padding: var(--spacing200) var(--spacing400);
+  box-sizing: border-box;
```

Comment:
looks like these css properties were not part of this component, and component will change visually after they are added. They should be added as separate commit, e.g.: `feat(drawer): add padding to sticky footer`